packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -243,7 +244,8 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
   /// This property can be set from a listener added to this
   /// [TextEditingController]; **however, one should not also set [selection]
   /// in a separate statement. To change both the [text] and the [selection]
-  /// change the controller's [value].**
+  /// change the controller's [value].** Setting this here will clear
```

Comment:
nit: I think most readers will interpret "clear" as "setting the current selection and composing range both to (0, 0)" but the selection range will actually be set to `(-1, -1)` which usually isn't what people wanted or expected.

Reply:
Maybe ‘invalidate’? Though I can see how that might be confusing too.